src/main/java/com/example/coursework2/service/JavaQuestionService.java

```diff
@@ -0,0 +1,60 @@
+package com.example.coursework2.service;
+
+import com.example.coursework2.Question;
+import com.example.coursework2.exeption.QuestionNumberException;
+import com.example.coursework2.interfaces.QuestionService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+@Service
+public class JavaQuestionService implements QuestionService {
+
+    private static final List<Question> questionList = new ArrayList<>();
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final List<Question> questionList = new ArrayList<>();
    private final Random random = new Random();
```
зачем static? не должны быть поля static